bindings/ergo-lib-wasm/src/wallet.rs

```diff
@@ -76,3 +77,9 @@ impl Wallet {
             .map(Transaction::from)
     }
 }
+
+impl From<ergo_lib::wallet::Wallet> for Wallet {
```

Comment:
`From` can be derived. See https://github.com/ergoplatform/sigma-rust/blob/ed132f81eaa612b39da3c5c180f244b458522916/bindings/ergo-lib-wasm/src/transaction.rs#L29-L30